applications/nrf5340_audio/src/bluetooth/Kconfig.defaults

```diff
@@ -0,0 +1,164 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+config BT_AUDIO
+	bool
+	default y
+
+# Not supported on Packetcraft controller
+config BT_ATT_ENFORCE_FLOW
+	bool
+	default n
+
+# Disable Zephyr HCI Vendor-Specific extensions
+config BT_HCI_VS_EXT
+	bool
+	default n
+
+# Prevent the MAC address from changing after every reboot
+config BT_PRIVACY
+	bool
+	default n
+
+config BT_EXT_ADV
+	bool
+	default y
+
+config BT_ISO_RX_MTU
+	int
+	default 190
```

Comment:
This defaults to 251, should we perhaps try with that and see what happens? Same with TX

Reply:
Maybe it is smart to do this in a separate PR?
I want to keep functional changes to a minimum in this PR.